guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -7,6 +7,11 @@ You must have a Red{nbsp}Hat subscription manifest containing a subscription all
 All subscription information is available in your Red Hat Customer Portal account.
 
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat subscription manifest in the Customer Portal.
+Manifests can only be created on the Customer Portal by {Project} users on a disconnected network.
+Manifests are created by {Project} users on a connected network using the Red{nbsp}Hat Hybrid Cloud Console.
+ifdef::satellite[]
+For more information on creating manifests on a connected network, see https://access.redhat.com/documentation/en-us/subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
```

Comment:
```suggestion
For more information on creating manifests on a connected network, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
```
We have [attributes for RHDocs](https://github.com/theforeman/foreman-documentation/blob/b807532d484dc40f299866459a9825156adae280/guides/common/attributes-base.adoc#L24) now.